src/rules_lastnum.txt

```diff
@@ -20,7 +20,9 @@ last num: number : + - [digits]
 22
 -986
 //
+
```

Comment:
can we clean up the diff by removing these empty lines?

Reply:
Yes, done.